frontend/class-opengraph.php

```diff
@@ -772,6 +787,19 @@ private function get_opengraph_image() {
 	}
 
 	/**
+	 * Check if taxonomy has an image and add this image
+	 *
+	 * @return bool
+	 */
+	private function get_opengraph_image_taxonomy() {
+		if ( ( $ogimg = WPSEO_Taxonomy_Meta::get_meta_without_term( 'opengraph-image' ) ) !== '' ) {
+			$this->add_image( $ogimg );
```

Comment:
Why are you returning 'true' here? The only time you call this class, is on line 734, but you don't use the return value.